test/jdk/java/nio/file/Files/CheckPermissions.java

```diff
@@ -219,6 +219,14 @@ public static void main(String[] args) throws IOException {
             exists(file);
```

Comment:
Sorry for the poor comment placement.  Do you need to add the bugid back at line 25?

Reply:
Thanks for the reminder and all set.